src/app/home-page/widget/entries/entries.component.html

```diff
@@ -0,0 +1,31 @@
+<div class="w-100">
+  <h3>My Entries</h3>
+  <div *ngIf="!hasEntries; else userHasEntries">
+    <p>You have no entries</p>
+  </div>
+
+  <ng-template #userHasEntries>
+    <mat-form-field class="w-100">
+      <input matInput [(ngModel)]="entryFilterText" (keyup)="onTextChange($event)" placeholder="Filter entries..." />
+    </mat-form-field>
+
+    <div *ngFor="let entry of myEntries; let i = index" class="w-100">
```

Comment:
Do you need `let i = index`? I don' think you use it, although I just scanned the snippet in here GitHub and may have overlooked it

Reply:
Good catch, I think I had tried something with it but then took it out.